scripts/ExportScripts/export_profile_monolithic.py

```diff
@@ -0,0 +1,55 @@
+from o3de.export_project import *
+from o3de import manifest
+import argparse
+import pathlib
+import os
+
+#Note: project_path and engine_path are computed before this script runs, and can be accessed via o3de_context.project_path
+#for now this script assumes an engine-centric approach, and we assume to use the source built engine for the engine_path
+
+#to use this script, go to the engine root directory, and issue the following command:
+#.\scripts\o3de.bat export-project -pp C:\path\to\project -es C:\path\to\export_profile_monolithic.py -nmp C:\path\to\build\non_mono -mp C:\path\to\build\mono -out C:\path\to\output -ll INFO
```

Comment:
Is there a way to place this script inside the scrtips/ folder and run it directly?

I am thinking that it would be good to break it up into smaller pieces and then combine different pieces together here.

Reply:
I think @AMZN-Olex means to run it directly using python.
Such as
```powershell
D:\o3de>python\python.bat scripts\ExportScripts\export_profile_monolithic.py ...
```

Reply:
I know @AMZN-alexpete recommend in an earlier PR that the `__main__` logic be removed from the export_project.py script, but all of our other o3de CLI scripts were setup to be run in a standalone fashion
For example the user can run the register.py script without going through the o3de.py driver (https://github.com/o3de/o3de/blob/development/scripts/o3de/o3de/register.py#L1053-L1075).

```
D:\o3de>python\python.bat scripts\o3de\register.py --this-engine
```

The only caveat with running the scripts in a standalone fashion, is that it requires to use the python that is installed inside of `<engine-root>\python` folder as we register the scripts\o3de package as a site-package as well as other requirements.